### ISSUE TITLE
Allow inserting descriptions of the pipelines

### ISSUE THREAD

**Issue by MrHedmad** [MrHedmad/kerblam#27]
To differentiate between the pipes one has only the filename to turn to. Since you write the filename each time you `kerblam run`, it is usually kept short.
A short name does not tell much about what a pipe does, so adding a description is preferable.

To do this, I propose adding a custom header in the makefiles/shellfiles. The first contiguous block of lines starting with just whitespace and `#?` is treated as a "docstring". For instance, in the `world.sh` file:

```bash
set -e
echo "Hello, world!"
#? Say hello to the world, then dominate it.
#?
#? This script is generally very simple as it just uses
#? an LLM to produce nonsense.

world --dominate
#? Disregard me.
```

The block:
```
Say hello to the world, then dominate it.

This script is generally very simple as it just uses
an LLM to produce nonsense.
```
will be the description of the file.

Running `kerblam run` will then display:
```
Available pipes:
	world 		Say hello to world, then dominate it.
    save        Save a copy of this text
    fast        Run faster than normal
```
Like a `git` commit the first line is the short version of the description (to be truncated if too long), while the full text is the full description.

To be decided is how to show the full description to the user (`kerblam run world --help`? `--describe`? `kerblam describe world`?).

**Comment by MrHedmad** [comment]
This was partially fixed in #41, but there is currently no way of showing the long description.

**Comment by MrHedmad** [comment]
Fixed by #50
